app/views.py

```diff
@@ -484,10 +484,6 @@ def post(self, request: HttpRequest, submission_id: int) -> HttpResponse:
 def ask_about_removing_post(request: HttpRequest, submission: Submission) -> None:
     """Ask Slack if we want to remove a reported submission or not."""
     # created using the Slack Block Kit Builder https://app.slack.com/block-kit-builder/
-    if request.GET.get("reason") is None:
-        # they just wanted to return the post. No reason to investigate.
-        return
```

Comment:
Why is this removed?
If they just want a different pot we shouldn't get a message on Slack

Reply:
Ah nevermind, I see it now. I suppose we don't need to construct the Slack blocks in that case, but it should be fine.